website/docs/src/pages/advanced-usage/components.mdx

```diff
@@ -504,6 +504,40 @@ export default () => (
 `}
 </CodeBlock>
 
+
+### Hiding Tests
+
+You can hide the test files with `visibleFiles` prop.
+Additionally if you want to supress test content and only show the test results, you can use `hideTestsAndSupressLogs` option.
+This option will hide the test files and supress the console logs. And it will disable the verbose button.
+
+<CodeBlock stack>
+{`import { 
```

Comment:
Can you lint/prettier this snippet?

Reply:
Sure! I ran it and fixed the issues you mentioned about linting & formatting. Also changed `hide` to `show` and removed `hideTestsAndSupressLogs` from context and added it to the test component. When I format it somehow messed up with tables, but just reverted back and added my change after.